pages/api/bitbucket/callbacks/webhook.ts

```diff
@@ -0,0 +1,26 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import { publishMessage } from '../../../../utils/pubsub/pubsubClient';
+import { getTopicNameFromDB } from '../../../../utils/db/relevance';
+
+const handler = async (req: NextApiRequest, res: NextApiResponse) => {
```

Comment:
Please name the function better. What is this handling? Which call? When does this get called? I have no idea what am I looking at except that this is called by Bitbucket as a provided callback function - which has something to do with webhook.